.github/matrix-jvm-tests.json

```diff
@@ -13,8 +13,8 @@
   "os-name": "ubuntu-latest"
 }
 , {
-  "name": "18",
-  "java-version": 18,
+  "name": "19",
+  "java-version": 19,
```

Comment:
It probably won't work because of Gradle.

Reply:
That's why I have the PR in draft. I'm waiting for CI on my fork

Reply:
Great, thanks!

Reply:
Indeed CI fails on some Gradle stuff. Seems we need to wait for Gradle 7.6 for this bump.